pandas/tests/tseries/test_offsets.py

```diff
@@ -41,6 +42,20 @@
 from pandas.tseries.holiday import USFederalHolidayCalendar
 
 
+offset_classes = [getattr(offsets, x) for x in dir(offsets)]
```

Comment:
no, pls just use the fixture (you can certainly create more if needed)

Reply:
OK.